recovery/cmd/launchBackup.go

```diff
@@ -181,6 +197,22 @@ func InitBackup() error {
 		return err
 	}
 
+	// Delete /run/ostree-booted file if empty
+	ostreeBooted := "/run/ostree-booted"
+	if _, err := os.Stat(ostreeBooted); err == nil {
+		if info, err := os.Stat(ostreeBooted); err == nil {
```

Comment:
Hmm - why do we need both these lines?

Reply:
Missed some cleanup :) This has changed...